tc_messages.js

```diff
@@ -24,12 +25,13 @@ module.exports.errors = {
  */
 module.exports.messages = {
 
-    'createdNewTimeCard':   "\n  "+chalk.bgCyan.black("TIMECARD:") + chalk.bold.white(" You have created a new timecard file. \n")+ chalk.bold.white("  Tip:") +" Clock in with the "+chalk.cyan("clockin")+" command, Clock out with the "+chalk.cyan("clockout")+" command \n",
-    'successfulClockin':    "\n  "+chalk.bgCyan.black("TIMECARD:") + chalk.bold.white(" You have ") + chalk.bold.green("clocked in: ") + time() + "\n",
-    'successfulClockout':   "\n  "+chalk.bgCyan.black("TIMECARD:") + chalk.bold.white(" You have ") + chalk.bold.red("clocked out: ") + time() + "\n",
-    'prettyPrintHeader':    "\n  "+chalk.bgCyan.black("TIMECARD:") + chalk.cyan(" Logged hours") + "\n\n  "+chalk.gray("______________________________________________\n"),
-    'prettyPrintBorder':    chalk.gray("\n  ______________________________________________")
-
+    'createdNewTimeCard':   	"\n  "+chalk.bgCyan.black("TIMECARD:") + chalk.bold.white(" You have created a new timecard file. \n")+ chalk.bold.white("  Tip:") +" Clock in with the "+chalk.cyan("clockin")+" command, Clock out with the "+chalk.cyan("clockout")+" command \n",
+    'successfulClockin':    	"\n  "+chalk.bgCyan.black("TIMECARD:") + chalk.bold.white(" You have ") + chalk.bold.green("clocked in: ") + time() + "\n",
+    'successfulClockout':   	"\n  "+chalk.bgCyan.black("TIMECARD:") + chalk.bold.white(" You have ") + chalk.bold.red("clocked out: ") + time() + "\n",
+    'prettyPrintHeader':    	"\n  "+chalk.bgCyan.black("TIMECARD:") + chalk.cyan(" Logged hours") + "\n\n  "+chalk.gray("______________________________________________\n"),
+    'prettyPrintBorder':    	chalk.gray("\n  ______________________________________________"),
+    'alreadExistsInGitIgnore':	"\n  "+chalk.bgCyan.black("GITIGNORE:") + chalk.bold.white(" The .gitignore file already contains .timecard.json. Nothing was appended! \n"),
```

Comment:
I think I prefer seeing nothing happen if there is no operation, much like the typical UNIX response you get from the command line: if nothing happens, you don't get response.